src/Universum/Monad/Reexport.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE CPP         #-}
 {-# LANGUAGE Trustworthy #-}
 
+-- | This module reexports functionality regarding monads.
```

Comment:
I was thinking on *functionality regarding smth* phrase and I feel something not quite right about that. Probably it's better to say *functions to work with smth*? If you agree probably replace all *regarding* words with other words?